torch/linalg/__init__.py

```diff
@@ -88,6 +88,49 @@
 Alias of :func:`torch.det`.
 """)
 
+matrix_rank = _add_docstr(_linalg.linalg_matrix_rank, r"""
```

Comment:
This needs to update the rst, too: 

https://github.com/pytorch/pytorch/blob/master/docs/source/linalg.rst